tsconfig.json

```diff
@@ -14,14 +15,18 @@
     "esModuleInterop": true,
     "noImplicitOverride": true,
     "noImplicitAny": false,
-    "outFile": "dist/types/index.d.ts"
+    "outFile": "dist/types/index.d.ts",
+    "module": "NodeNext",
+    "moduleResolution": "nodenext"
   },
   "include": [
     "src/api/**/*.js"
   ],
   "exclude": [
+    "e2e",
     "node_modules",
     "dist",
-    "**/*Spec.js"
+    "**/*Spec.js",
+    "openmct/e2e"
```

Comment:
now that we've confirmed that the working directory running `tsc` is the correct one, we don't need to exclude `openmct/e2e`